test/fixtures/o-test/.eslintrc

```diff
@@ -0,0 +1,38 @@
+{
+  "ecmaFeatures": {
+    "modules": true
+  },
+  "env": {
+    "es6": true,
+    "browser": true
+  },
+  "rules": {
+    "eqeqeq": 2,
+    "guard-for-in": 2,
+    "no-extend-native": 2,
+    "wrap-iife": 2,
+    "new-cap": 2,
+    "no-caller": 2,
+    "no-irregular-whitespace": 1,
+    "no-trailing-spaces": 2,
+    "no-multi-str": 0,
+    "dot-notation": 0,
+    "strict": [2, "function"],
+    "quotes": [1, "single"],
+    "valid-jsdoc": 1,
+    "no-multi-spaces": 2,
+    "no-multiple-empty-lines": 2,
```

Comment:
Interesting, I hadn't seen lintspaces before.  Does it provide features/rules which ESLint does not?